src/pybind/mgr/nfs/export.py

```diff
@@ -76,6 +77,16 @@ def validate_cephfs_path(mgr: 'Module', fs_name: str, path: str) -> None:
         raise NFSException(e.args[1], -e.args[0])
 
 
+def _validate_cmount_path(mgr: 'Module', fs_name: str, cmount_path: str, path: str) -> None:
+    validate_cephfs_path(mgr, fs_name, cmount_path)
+    if len(cmount_path) > len(path):
+        raise ValueError(
```

Comment:
This would then need to be separated for both the `path` and `cmount_path` validations